2_simple_parameters/README.md

```diff
@@ -1,8 +1,8 @@
-# Send data to LiveObjects
+# LiveObjects commands

```

Comment:
should be 'parameters'

Reply:
Also good point, thanks for watchfulness!